strawberryfields/backends/fockbackend/backend.py

```diff
@@ -322,8 +323,15 @@ def prepare_gkp(
         if representation == "complex":
             raise NotImplementedError("The complex description of GKP is not implemented")
 
-        if shape != "square":
-            raise NotImplementedError("Only square GKP are implemented for now")
+        if shape not in ["square", "rectangular"]:
+            raise NotImplementedError("Only square and rectangular GKP are implemented.")
+
+        if shape == "square":
+            if alpha != 1:
+                raise ValueError(
+                    "For square GKPs, alpha must be 1. For alpha not equal to "
+                    + "1, use shape='rectangular'."
+                )
 
         theta, phi = state[0], state[1]
-        self.circuit.prepare_gkp(theta, phi, epsilon, ampl_cutoff, self._remap_modes(mode))
+        self.circuit.prepare_gkp(theta, phi, epsilon, ampl_cutoff, alpha, self._remap_modes(mode))
```

Comment:
Possible argument order mismatch: in this argument list, `alpha` follows `ampl_cutoff`, whereas in circuit.prepare_gkp `alpha` follows `epsilon`.

I think this ends up fixing itself by accident since the order in `ops.rect_gkp_state` is again flipped but it should definitely be changed!

Reply:
Great catch!